docs/crashlytics/usage.mdx

```diff
@@ -3,15 +3,47 @@ title: Using Firebase Crashlytics
 sidebar_label: Usage
 ---
 
+## Getting started
+
+Enable Firebase Crashytlics in the Firebase console.
+
+![Crashytlics - Enable in the console](enable-firebase-crashlytics.png)
+
+Until an error has been reported, you will see this screen.
+
+![Crashytlics init screen](crashlytics-init.png)
+
 To start using Firebase Crashlytics within your project, import it at the top of your project files:
 
 ```dart
 import 'package:firebase_crashlytics/firebase_crashlytics.dart';
 ```
 
+Run an example, such as...
+
+```dart
+// Init application
+await Firebase.initializeApp();
+
+// Elsewhere in your code
+FirebaseCrashlytics.instance.crash();
+```
+
+This will restart the application. If using an emulator, please re-run:
+
+```bash
+  flutter run
+```
+
+`or restart the app manually through the emulator.`
```

Comment:
```suggestion
This will crash the currently running application. You will then need to manually re-open your application on your emulator for Crashlytics to submit the crash report to the Firebase Console.
```